### PR TITLE
feat(presenter): typecheck in vite

### LINKED ISSUE #42: 🐛 [Bug] String types are not properly checked in vue3 components
## 🐛 Bug

String types are not properly checked in vue3 components.

Example is the LogoImage - Size.

### PR DESCRIPTION
**Pull request by ulfgebhardt**
## 🍰 Pullrequest

typecheck in vite

### Issues

- fixes #42 
- this PR also fixes the local performance problem - if the remote performance was fixed is unclear

### Todo

- [X] None

### CODE REVIEW

File: presenter/src/components/inputs/MainButton.test.ts [hunk 1]
```diff
@@ -46,30 +45,34 @@ describe('MainButton', () => {
...
+        await wrapper.setProps({ label: 'My Button', variant: 'primary', size: 'small' })
         expect(wrapper.find('.v-btn').classes()).toContain('main-button-small')
       })
     })
 
     // needed to catch the || branch
     describe('size empty string', () => {
       it('sets class main-button-medium', async () => {
-        await wrapper.setProps({ size: '' })
+        await wrapper.setProps({ label: 'My Button', variant: 'primary', size: '' })
```
Comment (ulfgebhardt) [inline presenter/src/components/inputs/MainButton.test.ts:56]: this fails now

File: presenter/src/components/inputs/MainButton.test.ts [hunk 2]
```diff
@@ -46,30 +45,34 @@ describe('MainButton', () => {
...
+        await wrapper.setProps({ label: 'My Button', variant: 'primary', size: undefined })
         expect(wrapper.find('.v-btn').classes()).toContain('main-button-medium')
       })
     })
 
     // is this behaviour wanted? Do we have to improve this?
     describe('size is some weird string', () => {
       it('sets strange classes', async () => {
-        await wrapper.setProps({ size: 'some weird string' })
+        await wrapper.setProps({
```
Comment (ulfgebhardt) [inline presenter/src/components/inputs/MainButton.test.ts:71]: this fails now

**Comment by ulfgebhardt** [comment]
image

There is actually three tests failing.

**Review by Mogge — DISMISSED** [review]
Great! Thanks that you looked after that. I like the new typedef of  the wrapper

**Review by Mogge — APPROVED** [review]

**Pull request merged**